channels/us.m3u

```diff
@@ -2386,12 +2380,6 @@ https://mhz-samsung-linear-ca.samsung.wurl.com/manifest/playlist.m3u8
 https://mhz-samsung-linear.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="MiamiCityTV.us" tvg-name="Miami City TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/HJohrzO.png" group-title="Local",Miami City TV (360p)
 https://granicusliveus9-a.akamaihd.net/miamifl/G2076_003/playlist.m3u8
-#EXTINF:-1 tvg-id="MiamiTV.us" tvg-name="Miami TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/WlrcyMe.png" group-title="XXX",Miami TV
-https://59ec5453559f0.streamlock.net/miamitv/smil:miamitvROKU/chunklist_w1195980091_b2592000.m3u8
```

Comment:
miamitvROKU, can be alternate variant

Reply:
Yup, there's a playlist below it.

Reply:
that is miamitvWEB

Reply:
Welp, my bad. Got up with the mexican link that had "Roku" in his name.

Reply:
Fast patched up on his PR.